test/unit/spec/MfaVerify_spec.js

```diff
@@ -1585,6 +1615,37 @@ function (Okta,
             expect(test.form.errorBox().length).toBe(0);
           });
       });
+
+      itp('on WARNING_TIMEOUT, after dail/re-dial, shows warning message for call after 30s idle', function () {
```

Comment:
text should be just shows warning message to click "Redial" after 30s

Reply:
fixed